tools/autograd/derivatives.yaml

```diff
@@ -2982,3 +2982,30 @@
 
 - name: special_spherical_bessel_j0(Tensor x) -> Tensor
   x: non_differentiable
+
+- name: special_cos_pi(Tensor z) -> Tensor
```

Comment:
@albanD These operations are obviously differentiable but I plan on revisiting special function derivatives once we have a fuller suite of special functions to use.